src/main/java/com/feelmycode/parabole/dto/EventApplyDto.java

```diff
@@ -0,0 +1,34 @@
+package com.feelmycode.parabole.dto;
+
+import com.feelmycode.parabole.domain.Event;
+import com.feelmycode.parabole.domain.EventParticipant;
+import com.feelmycode.parabole.domain.EventPrize;
+import com.feelmycode.parabole.domain.User;
+import java.time.LocalDateTime;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+
+@Getter
+public class EventApplyDto {
+
+    @NotNull
+    private Long userId;
+    @NotNull
+    private Long eventId;
+    @NotNull
+    private Long prizeId;
+    private String participantAt;
+
+    public EventApplyDto(){
+
+    }
```

Comment:
기본 선언자가 내부가 띄어질 필요는 없을 거 같아요.
아래처럼 한줄 넘기거나 아니면 아예 안넘겨도 될거같아요
```suggestion
    public EventApplyDto(){
    }
```